### ISSUE TITLE
Official code snap is behind the latest update

### ISSUE THREAD

**Issue by tristanHessell** [Microsoft/vscode#72180]
The official code snap isn't the update.

Snapcrafters vscode: https://snapcraft.io/vscode (Current is `latest/stable 1.33.0-1554390824`)
Visual Studio Code code: https://snapcraft.io/code (Current is `latest/stable 51b0b281`)

`51b0b281` correlates to version 1.32.3.

I've pasted my full details below (from the `About` modal).
```
Version: 1.32.3
Commit: a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4
Date: 2019-03-14T23:38:49.842Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.18.0-16-generic
```

**Comment by joaomoreno** [comment]
The latest released build from us is `51b0b281`, which is indeed already in the snap store: https://snapcraft.io/code

Snapcrafters vscode is an unnoficial distribution, not coming from us.

**Comment by reitzig** [comment]
@joaomoreno I don't know, this is what it looks like for me:

![image](https://user-images.githubusercontent.com/4246780/57129536-02719c00-6d97-11e9-8bbf-6f3866448a12.png)

And code keeps nagging me to download an update I can't install by clicking "download now", anyway (it should never show that notification in a snap install, I think, but that's another issue).

**Comment by joaomoreno** [comment]
Can you run `snap info code`? This is what I get:

```
$ snap info code 
name:      code
summary:   Code editing. Redefined.
publisher: Visual Studio Code (vscode✓)
contact:   https://twitter.com/code
license:   unset
description: |
  Visual Studio Code is a lightweight but powerful source code editor which runs on your desktop and
  is available for **Linux**, **macOS** and **Windows**. It comes with built-in support for
  **JavaScript**, **TypeScript** and **Node.js** and has a rich ecosystem of extensions for other
  languages (such as **C++**, **C#**, **Java**, **Python**, **PHP**, **Go**) and runtimes (such as
  **.NET** and **Unity**).
  
  By downloading and using Visual Studio Code, you agree to the license terms
  (https://code.visualstudio.com/License/) and privacy statement
  (https://privacy.microsoft.com/en-us/privacystatement). Visual Studio Code automatically sends
  telemetry data and crash dumps to help us improve the product. If you would prefer not to have
  this data sent please go see
  https://code.visualstudio.com/docs/supporting/FAQ#_how-to-disable-crash-reporting to learn how to
  disable it.
  
  https://github.com/Microsoft/vscode
commands:
  - code
  - code.url-handler
snap-id:      Ht0aUHi7ofh9Fbwh6m7jUN2pAy6kzBiu
tracking:     stable
refresh-date: 21 days ago, at 03:02 CEST
channels:
  stable:    51b0b281 2019-04-11 (6) 133MB classic
  candidate: ↑                             
  beta:      ↑                             
  edge:      ↑                             
installed:   51b0b281            (6) 133MB classic
```

**Comment by reitzig** [comment]
Same:

```
commands:
  - code
  - code.url-handler
snap-id:      Ht0aUHi7ofh9Fbwh6m7jUN2pAy6kzBiu
tracking:     stable
refresh-date: 8 days ago, at 17:19 CEST
channels:
  stable:    51b0b281 2019-04-11 (6) 133MB classic
  candidate: ↑                             
  beta:      ↑                             
  edge:      ↑                             
installed:   51b0b281            (6) 133MB classic
```

Oh, derp:

```
$ aptitude show code
Package: code                            
Version: 1.33.1-1554971066
New: yes
State: installed (1.33.0-1554390824), upgrade available (1.33.1-1554971066)
Automatically installed: no
```

I guess that's what it starts quickly, too. X) Never mind, sorry for the fuzz. I'll show myself out.
